source/docs/api-reference/api-usage/signal-logging.rst

```diff
@@ -168,6 +170,35 @@ Any log that contains a :doc:`pro-licensed </docs/licensing/licensing>` device w
    - Yaw
    - AngularVelocityZWorld
 
+   **CANrange:**
+
+   - SupplyVoltage
+   - DistanceMeters
+   - ProximityDetected
+   - SignalStrength
+
+   **CANdi:**
+
+   - SupplyVoltage
+   - Pin1State
+   - Pin2State
+   - S1Closed
+   - S2Closed
+   - QuadPosition
+   - QuadVelocity
+   - Pwm1_Position
+   - Pwm1_Velocity
+   - Pwm2_Position
+   - Pwm2_Velocity
+   - Overcurrent
+
+   **Talon FXS:**
+
+   - ConnectedMotor
+   - Fault_HallSensorMissing
+   - Fault_DriveDisabledHallSensor
+   - Fault_MotorTempSensorMissing
```

Comment:
This is kind of confusing in that it doesn't include all the Talon FX signals.

Reply:
Sure.